ingest/rules/nextclade.smk

```diff
@@ -45,11 +45,36 @@ rule run_nextclade:
         """
 
 
+rule nextclade_metadata:
+    input:
+        nextclade="results/nextclade.tsv",
+    output:
+        nextclade_metadata=temp("results/nextclade_metadata.tsv"),
```

Comment:
Is there a reason to mark this as `temp`? Would it not be useful for debugging along with all the other intermediate files in `results/`?

Reply:
It's marked as temp because the next rule (`join_metadata_and_nextclade`) merges this file and the other metadata (`data/subset_metadata.tsv`) together, so all the data ends up in that file anyway. Any debugging would most likely happen using that file.

Reply:
What if we want to debug `augur curate rename` in this rule and `join_metadata_and_nextclade` fails or we simply don't want to run it?

My argument is mostly based on consistency, since the same could be said about `results/nextclade.tsv`: why not make that temp because it eventually makes its way into `results/metadata.tsv`?

Consider this non-blocking since it works fine.

Reply:
> What if we want to debug `augur curate rename` in this rule and `join_metadata_and_nextclade` fails or we simply don't want to run it?

In such a situation, Snakemake's `--notemp` option could be used.

Reply:
> My argument is mostly based on consistency, since the same could be said about `results/nextclade.tsv`: why not make that temp because it eventually makes its way into `results/metadata.tsv`?

`results/nextclade.tsv` and `results/nextclade_metadata.tsv` are effectively isotopes of the same underlying data; we don't need to retain both of them, because if you have one, you can tell what had to have been in the other one. I marked the latter one as temp because it's the newer of the two, so it was less change.